### PR TITLE
Use more common 'qw()' style

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by richiejp** [comment]
Looks fine apart from merge conflicts.

**Comment by okurz** [comment]
let's wait for https://github.com/os-autoinst/os-autoinst/pull/655 with a better policy attached, then we will update the tests

File: lib/mm_network.pm [hunk 1]
```diff
@@ -7,7 +7,7 @@ use Exporter;
 
 use testapi;
 
-our @EXPORT = qw/configure_hostname get_host_resolv_conf configure_static_ip configure_dhcp configure_default_gateway configure_static_dns parse_network_configuration ip_in_subnet check_ip_in_subnet/;
+our @EXPORT = qw(configure_hostname get_host_resolv_conf configure_static_ip configure_dhcp configure_default_gateway configure_static_dns parse_network_configuration ip_in_subnet check_ip_in_subnet);
```
Comment (coolo) [inline lib/mm_network.pm]: I guess we should sync the perltidy settings as well - the lines are much longer here

**Comment by coolo** [comment]
I don't see that policy

**Comment by okurz** [comment]
please look again. perltidyrc, will see

**Pull request merged**